app/lib/ui/pr_card.dart

```diff
@@ -103,13 +103,7 @@ List<Widget> cardContentForSize(BuildContext context, PR pr) {
                       overflow: TextOverflow.ellipsis,
                       style: Theme.of(context).textTheme.overline,
                     ),
-                    Text(
-                      DateFormat("dd MMM. yyyy").format(pr.created) +
-                          " by " +
-                          pr.user,
-                      overflow: TextOverflow.ellipsis,
-                      style: Theme.of(context).textTheme.overline,
-                    ),
+                    conditionallyColoredDuration(context, pr.created)
```

Comment:
I'm sry but I'm still not completely happy with this view.
![grafik](https://user-images.githubusercontent.com/39572273/162370955-b497fd9a-96c4-43b6-a672-9a7a11d9d4ea.png)
I feel like the timestamp has too much priority now. It almost looks like the timestamp is more important than the PR's title.

I think there are multiple possibilities
1. Only put the text here like before (but formatted with `timeago`). No color coding then.
2. Try other more subtle ways of color coding to not make the timestamp too big. Like maybe an underline or sth but not sure about this one 😄 
3. Just keep it like this, noone ever uses the app in portrait mode and maybe the time stamp is actually much more important than the title